numeral.js

```diff
@@ -394,8 +411,7 @@
             if (format.indexOf('.') === 0) {
                 w = '';
             }
-
-            return ((negP && neg) ? '(' : '') + ((!negP && neg) ? '-' : '') + ((!neg && signed) ? '+' : '') + w + d + ((ord) ? ord : '') + ((abbr) ? abbr : '') + ((bytes) ? bytes : '') + ((negP && neg) ? ')' : '');
+            return ((negP && neg) ? '(' : '') + ((!negP && neg) ? '-' : '') + ((!neg && signed) ? '+' : '') + w + d + ((ord) ? ord : '') + ((abbr && !omitAbbr) ? abbr : '') + ((bytes && !omitAbbr) ? bytes : '') + ((negP && neg) ? ')' : '');
```

Comment:
If it were just about my choice, I would also turn the overuse of ?: into a series of if/then/else.

But we do want to minimize churn so we can get this merged into mainline with a bit less fuss.